Fixed Bugs

no1mann opened the pull request:
Fixed (Hack v1.1.0):
- Fixed most mid importing bugs
- If Mid fails to import, will revert back to original GHTCP method
- Fixed maximum value for default unlocked songs from 100 to 1,000,000,000
- Texture Explorer now works (Fixed by Exilelord, not me)
- Fixed notification icon bug where it wouldn't disappear on close
- Fixed error when adding songs to setlists/tiers
- Default GH songs are now editable (export as chart/change settings)

no1mann committed:
Merge remote-tracking branch 'refs/remotes/ExileLord/master'

no1mann committed:
Merge remote-tracking branch 'refs/remotes/ExileLord/master'

no1mann committed:
Fixed Bugs

Fixed (Hack v1.1.0):
- Fixed most mid importing bugs
- If Mid fails to import, will revert back to original GHTCP method
- Fixed maximum value for default unlocked songs from 100 to
1,000,000,000
- Texture Explorer now works (Fixed by Exilelord, not me)
- Fixed notification icon bug where it wouldn't disappear on close

no1mann committed:
A couple final fixes

A couple final fixes

kolatra commented:
Making the default GH songs editable is great because people can stop complaining about Cult of Personality's name being strange and just fix it. (Although it could already be done in QueenBee, but not a lot of people cared enough to do that.)

no1mann committed:
Added a disclaimer for editing default GH3 songs

Also, I changed it so you still can't delete default songs because I
think you can guess why

no1mann committed:
Minor Change (Default unlocked is now 65536)

no1mann committed:
Merge remote-tracking branch 'refs/remotes/ExileLord/master'

no1mann committed:
Fixed "Hide Original Songs" option

Fixed "Hide Original Songs" option
